### DISCUSSION TITLE
Importing a map below selected difficulty range then clicking on notification doesn't lead to it

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#25025] [Q&A]
https://github.com/ppy/osu/assets/86934170/73e26be4-146d-4f4d-b53a-d57458a89dba

**Comment by Loreos7** [comment]
Bumping this up because no one has set a tag.

**Reply by Joehuu** [reply]
May be fixed in https://github.com/ppy/osu/pull/33012.
